src/test/java/com/commercetools/sync/commons/MockUtils.java

```diff
@@ -22,6 +40,91 @@ public static CustomFieldsDraft getMockCustomFieldsDraft() {
         customFieldsJsons.put("invisibleInShop", JsonNodeFactory.instance.booleanNode(false));
         customFieldsJsons
             .put("backgroundColor", JsonNodeFactory.instance.objectNode().put("de", "rot").put("en", "red"));
-        return CustomFieldsDraft.ofTypeKeyAndJson("StepCategoryTypeKey", customFieldsJsons);
+        return CustomFieldsDraft.ofTypeIdAndJson("StepCategoryTypeId", customFieldsJsons);
+    }
+
+    /**
+     * Creates a mock instance of {@link CustomFieldsDraft} with the supplied {@code customTypeId} and
+     * {@code customFields}.
+     *
+     * @return a mock instance of {@link CustomFieldsDraft} with supplied id and custom fields.
+     */
+    public static CustomFieldsDraft getMockCustomFieldsDraft(@Nullable final String customTypeId,
+                                                             @Nonnull final Map<String, JsonNode> customFields) {
+        final CustomFieldsDraft newCategoryCustomFieldsDraft = mock(CustomFieldsDraft.class);
```

Comment:
in this specific case actually it is better to use the builder for instantiation because I was trying to mock it specifically to be able to pass null ids for tests. Which is actually impossible to do when instantiating through the normal builder. Therefore I changed it here: 	37cf6ed and removed the impossible test cases. But in some other cases it is easier to mock using mockito for example to mock a category draft without a slug..